src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -342,6 +342,12 @@ class UserService(
         updatedAt = null,
       ),
     )
+    return GetUserResponse(savedUser, true)
+  }
+
+  fun getExistingUserOrCreate(username: String): UserEntity {
+    val getUserResponse = getExistingUserOrCreate(username, dontThrowExceptionOnStaffRecordNotFound = false)
+    return getUserResponse.user!!
```

Comment:
i think this can be flattened into a single line, removing the need for the curly braces and return type definition e.g.

```
fun getExistingUserOrCreate(username: String) = getExistingUserOrCreate(username, dontThrowExceptionOnStaffRecordNotFound = false).user!!
```